apps/academy/src/data/quizzes/nft-solidity/quiz-nft-solidity-5.json

```diff
@@ -0,0 +1,280 @@
+{
+  "title": "Quiz: Lesson 5",
+  "questions": [
+    {
+      "question": "In the lesson, what is the purpose of creating a new sub-directory named `frontend`?",
+      "options": [
+        {
+          "answer": "To speed up the development process"
+        },
+        {
+          "answer": "To maintain good coding practices",
+          "correct": true
+        },
+        {
+          "answer": "To hide code from the compiler"
+        }
+      ]
+    },
+    {
+      "question": "RainbowKit kit is used in our dApp setup. What is it?",
+      "options": [
+        {
+          "answer": "It’s responsible for the database management"
+        },
+        {
+          "answer": "It tracks the blockchain transactions"
+        },
+        {
+          "answer": "It's a testing framework"
+        },
+        {
+          "answer": "It's a library for wallet management",
+          "correct": true
+        }
+      ]
+    },
+    {
+      "question": "Why is the `useEffect` hook used in the pages.js file?",
+      "options": [
+        {
+          "answer": "To fix the hook if its code breaks"
+        },
+        {
+          "answer": "To expose the code from RainbowKit"
+        },
+        {
+          "answer": "To automatically update the contract's Solidity version when there's a breaking change"
+        },
+        {
+          "answer": "To manage changes and side effects from its own functions in the hook",
+          "correct": true
+        }
+      ]
+    },
+    {
+      "question": "We talk about the `TierABI` in the lesson. What is it?",
+      "options": [
+        {
+          "answer": "A React library that let’s us mint our NFT in tiers"
```

Comment:
I think it should be "lets us"

Reply:
haha, you've seen a lot of those 'let's' recently 🤣